db/seeds.rb

```diff
@@ -43,6 +43,7 @@
 
 p 'creating classroom'
 
+<<<<<<< HEAD
```

Comment:
missing merge conflict

Reply:
This merge conflict is also not present on my machine.